projection/core/src/main/resources/reference.conf

```diff
@@ -0,0 +1 @@
+lagom.projection.auto-start.enabled = true
```

Comment:
This is the current behaviour in Lagom. Projections start by default. 

Users can disable it and use Projections API to start them out.